src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const list = document.querySelector('ul');
+
+function sortList(listItems) {
+  const sortedList = [...listItems].sort((a, b) => {
+    const numA = toNumber(a);
+    const numB = toNumber(b);
+
+    return numB - numA;
+  });
+
+  // Remove all child elements from `list`
+  while (list.firstChild) {
+    list.removeChild(list.firstChild);
+  }
+
+  // Append the sorted elements to `list`
+  sortedList.forEach((item) => {
+    list.appendChild(item);
+  });
+
+  return list;
+}
+
+function toNumber(listItem) {
+  return listItem.dataset.salary.slice(1).split(',').join('');
+}
+
+function getEmployees(listItems) {
+  return [...listItems].map((item, index) => ({
```

Comment:
You can avoid using spreading array here. `map` method returns a new array always. So you can use just `return listItems.map(...)`